extensions/ql-vscode/src/databases/github-database-download.ts

```diff
@@ -66,40 +63,46 @@ export async function downloadDatabaseFromGitHub(
   cliServer: CodeQLCliServer,
   commandManager: AppCommandManager,
 ): Promise<void> {
-  const languages = databases.map((database) => database.language);
-
-  const language = await promptForLanguage(languages, undefined);
-  if (!language) {
+  const selectedDatabases = await promptForDatabases(databases);
+  if (selectedDatabases.length === 0) {
     return;
   }
 
-  const database = databases.find((database) => database.language === language);
-  if (!database) {
-    return;
-  }
+  await Promise.all(
+    selectedDatabases.map((database) =>
+      withProgress(
+        async (progress) => {
+          await downloadGitHubDatabaseFromUrl(
```

Comment:
I think it's hard to do that since the selection is not controlled by this function itself. It sounds sound like a good idea, but maybe we can see if there's any feedback about this from users and implement something else based on that.